.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+defaults: &defaults
+  working_directory: /home/circleci/grafana-kentik-app
```

Comment:
I think it's better to use relative path like `~/grafana-kentik-app`, because `/home/circleci` not guaranteed and can be changed in future.